src/index.js

```diff
@@ -21,15 +21,16 @@ import {
   errorCodeToString,
   getVersion,
   PAYLOAD_TYPE,
+  SCHEME,
   processErrorResponse,
 } from "./common";
 
 import { CLA, SLIP0044 } from "./config";
 
 const INS = {
   GET_VERSION: 0x00,
-  GET_ADDR_ED25519: 0x01,
-  SIGN_ED25519: 0x02,
+  GET_ADDR: 0x01,
```

Comment:
This change will break backward compatibility :(
we can add the new ones (which are cleaner) but 
we need to keep the old ones.. at least for now and add a comment `//deprecated`